Collaboration.md

```diff
@@ -0,0 +1,155 @@
+# Bitcoin Design Community collaboration playbook
+
+As we get actively involved in supporting projects, we will probably encounter a few different types of situations. The goal of this document is to define common types of collaborations, to serve as starting points for new inquiries.
```

Comment:
This intro sentence seems redundant to me.
```suggestion
```

Reply:
I find it important to clarify the purpose of the document. Curious what others think.

Reply:
Oh my bad didn't want to delete the purpose, just the first sentence.